src/components/button-toggle-group/button-toggle-group.component.js

```diff
@@ -72,4 +71,4 @@ BaseButtonToggleGroup.propTypes = {
 
 BaseButtonToggleGroup.displayName = 'BaseButtonToggleGroup';
 
-export default withValidations(BaseButtonToggleGroup);
+export default BaseButtonToggleGroup;
```

Comment:
There is no validations story for `Button Toggle Group`, please can you add one.